witness/src/Dscp/Witness/Launcher/Mode.hs

```diff
@@ -74,12 +78,12 @@ type WitnessWorkMode ctx m =
 
 -- | Context is resources plus some runtime variables.
 data WitnessContext = WitnessContext
-    { _wcParams     :: !WitnessParams     -- ^ Parameters witness was started with.
-    , _wcResources  :: !WitnessResources  -- ^ Resources, allocated from params.
-    , _wcMempool    :: !MempoolVar
-    , _wcSDActions  :: !SDActions
-    , _wcRelayState :: !RelayState
-    , _wcSDLock     :: !SDLock
+    { _wcParams          :: !WitnessParams     -- ^ Parameters witness was started with.
+    , _wcResources       :: !WitnessResources  -- ^ Resources, allocated from params.
+    , _wcMempool         :: !MempoolVar
+    , _wcSDActions       :: !SDVars
+    , _wcRelayState      :: !RelayState
+    , _wcSDLock          :: !SDLock
```

Comment:
Good news: these spaces are not needed.
Bad news: somebody still hasn't added `stylish-haskell` to save hook.